config.json

```diff
@@ -14,5 +14,9 @@
   "call_add_cards_hook": false,
   "enable_debug_log": false,
   "search_the_web": false,
-  "timeout_seconds": 60
+  "timeout_seconds": 60,
+  "show_extended_filters": true,
+  "show_help_buttons": true,
+  "import_source_info": false,
```

Comment:
This config key import_source_info appears unused. I assume it needs to be implemented in a different PR.